protos/flyteidl/core/workflow.proto

```diff
@@ -124,6 +129,10 @@ message Node {
 // Metadata for the entire workflow.
 // To be used in the future.
 message WorkflowMetadata {
+    // Identify whether workflow is interruptible.
+    // The value set at the workflow level will be the defualt value used for nodes
+    // unless explicitly set at the node level.
+    bool interruptible = 1;
```

Comment:
Can you rename it to reflect it's the default value? or add a WorkflowMetadataDefaults message and reference it here...
CC: @surindersinghp Isn't that what we talked about?

Reply:
yes, fixing.

Reply:
@EngHabu changing type name shouldn't break anything.

Reply:
Hmmm 🤞I hope that doesn't break the models code in the SDK